x-pack/plugin/esql/qa/server/multi-clusters/src/javaRestTest/java/org/elasticsearch/xpack/esql/ccq/Clusters.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.esql.ccq;
+
+import org.elasticsearch.test.cluster.ElasticsearchCluster;
+import org.elasticsearch.test.cluster.local.distribution.DistributionType;
+import org.elasticsearch.test.cluster.util.Version;
+
+public class Clusters {
+    public static ElasticsearchCluster remoteCluster() {
+        return ElasticsearchCluster.local()
+            .name("remote_cluster")
+            .distribution(DistributionType.DEFAULT)
+            .version(Version.fromString(System.getProperty("tests.old_cluster_version")))
+            .nodes(2)
+            .setting("node.roles", "[data,ingest,master]")
+            .setting("xpack.security.enabled", "false")
+            .setting("xpack.license.self_generated.type", "trial")
+            .build();
+    }
+
+    public static ElasticsearchCluster localCluster(ElasticsearchCluster remoteCluster) {
+        return ElasticsearchCluster.local()
+            .name("local_cluster")
+            .distribution(DistributionType.DEFAULT)
+            .version(Version.CURRENT)
```

Comment:
FWIW, specifying `Version.CURRENT` explicitly isn't necessary since this is the default. Probably doesn't hurt to be explicit there though given these are BWC tests with multiple versions.